.github/actions/mockTest/action.yml

```diff
@@ -1,6 +1,32 @@
-name: "Mock Test"
+name: "Mock Tests"
 description: "Run mock tests"
-
 runs:
-  using: "node12"
-  main: "./main.js"
+  using: "composite"
+  steps:
+    - name: Setup node
+      uses: actions/setup-node@v2
+      with:
+        node-version: "15"
+        cache: "npm" # - name: "Setup node"
+    - name: Install dependencies
+      run: npm ci
+      shell: bash
+    - name: Copy test files
+      run: cp -r test public/test
+      shell: bash
+    - name: "Setup NextJS Cache"
+      uses: actions/cache@v2
+      with:
+        path: ${{ github.workspace }}/.next/cache
+        # Generate a new cache whenever packages or source files change.
+        key: ${{ runner.os }}-nextjs-${{ hashFiles('**/package-lock.json') }}-${{ hashFiles('**.[jt]s', '**.[jt]sx') }}
+        # If source files changed but packages didn't, rebuild from a prior cache.
+        restore-keys: |
+          ${{ runner.os }}-nextjs-${{ hashFiles('**/package-lock.json') }}-
+    - run: ./node_modules/.bin/next dev -p 8080 &
```

Comment:
Nice using `&` here!

Reply:
Thanks! It felt a *little* hacky, but better than trying to do the `start a shell process from inside of the test runner` thing 🤷

Reply:
oh gosh - i thought we might need to background a process 🙍‍♂️